cajita/src/Cajita_DynamicPartitioner.hpp

```diff
@@ -29,14 +29,15 @@ namespace Cajita
 {
 //---------------------------------------------------------------------------//
 /*!
-  Sparse mesh block partitioner. (Current Version: Support 3D only)
-  \tparam Device Kokkos device type.
-  \tparam CellPerTileDim Cells per tile per dimension.
-  \tparam NumSpaceDim Dimemsion (The current version support 3D only)
+  Dynamic mesh block partitioner. (Current Version: Support 3D only) There
+  should be no instantiation for this class without implementing any workload
```

Comment:
This should be virtual then, right?

Reply:
Do you mean the function `setLocalWorkload()` should be virtual function? However, `ParticleDynamicPartitioner.setLocalWorkload()` and `SparseMapDynamicPartitioner.setLocalWorkload()` have different input parameters now. I have no idea how to deal with it.